Makefile

```diff
@@ -128,8 +128,9 @@ check-no-changes:
 ## check-python : Python Linting.
 .PHONY: check-python
 check-python: python-env
-	@$(FIND) -name *.py -name *.py -not -path "*/build/*" -exec $(PYTHON_ENV)/bin/autopep8 -d --max-line-length 120  {} \; | (! grep . -q) || (echo "Code differs from autopep8's style" && false)
-	@$(FIND) -name *.py -not -path "*/build/*" | xargs $(PYTHON_ENV)/bin/pylint --py3k -E || (echo "Code is not compatible with Python 3" && false)
+	@. $(PYTHON_ENV)/bin/activate; \
```

Comment:
Interesting that this is needed here, commands are executed with the full path under the venv.

Reply:
I wonder if it would be worth to "reasses" these lines as it seems there wer some challenges around it recently.